ydb/core/base/blobstorage.h

```diff
@@ -841,6 +841,17 @@ struct TEvBlobStorage {
         EvControllerGroupDecommittedNotify          = 0x1003161e,
         EvControllerGroupDecommittedResponse        = 0x1003161f,
         EvControllerGroupMetricsExchange            = 0x10031620,
+        EvControllerProposeConfigRequest            = 0x10031621,
+        EvControllerProposeConfigResponse           = 0x10031622,
+        EvControllerConsoleCommitRequest            = 0x10031623,
+        EvControllerConsoleCommitResponse           = 0x10031624,
+        EvControllerValidateConfigRequest           = 0x10031625,
+        EvControllerValidateConfigResponse          = 0x10031626,
+        EvControllerRetryConsoleCommit              = 0x10031627,
+        EvControllerValidationTimeout               = 0x10031628,
```

Comment:
it is internal message for the BSC, you don't need to put it to global scope (consider ES_PRIVATE)